pyspark_counter.py

```diff
@@ -0,0 +1,32 @@
+from pyspark import SparkContext
+from pyspark.streaming import StreamingContext
+from pyspark.streaming.kafka import KafkaUtils
+import json
+import argparse
+
+
+def main(topic_name, slide_interval=30, micro_secs_batch=10, output_file=None, checkpoint_file='/tmp/spark_twitter'):
+    sc = SparkContext(appName="PythonSparkStreamingKafka_RM_01")
+    sc.setLogLevel("WARN")
+
+    ssc = StreamingContext(sc, micro_secs_batch)
+    ssc.checkpoint(checkpoint_file)
```

Comment:
Verify with a test how this checkpoint work: run the spark application, and then make it failed some worker (turn it off) and the start it again, and check out how this work